aws-ssm-parameter/aws-ssm-parameter.json

```diff
@@ -78,15 +78,18 @@
     "primaryIdentifier": [
         "/properties/Name"
     ],
-    "readOnlyProperties": [
-        "/properties/Name",
-        "/properties/Type",
-        "/properties/Value"
+    "writeOnlyProperties": [
+        "/properties/Tags",
```

Comment:
WriteOnly usually just apply to sensitive data, why we mark these properties as WriteOnly here?

Reply:
Hi because the FAS policy we have only allows us to use getParameter API to get information for read handler. getParameter API will not return these properties so these properties will be added to writeOnlyProperties and will not be returned in read handler. Discussed with Himanshu Srivastava.